### ISSUE TITLE
Client Side Exporting is raising an error in IE11

### ISSUE THREAD

**Issue by hudson-gev-art** [highcharts/highcharts#14954]
#### Expected behaviour

The Client Side Exporting of PNG/JPG should work in IE11 when including the exporting and offline-exporting modules.

#### Actual behaviour

When going to download, get the error message: "Object doesn't support property or method 'remove'"

#### Live demo with steps to reproduce
Unfortunately it is an IE11 issue and jsFiddle doesn't support. This is the [feature example](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/exporting/offline-download-demo).

I think the issue is related to this [change](https://github.com/highcharts/highcharts/pull/14097/files#diff-bcca82be8ec262727e0d70b38b4c0dfcd827c11bd3f355aa5f5e90161ff2e87dR1875). IE11 doesn't support the method remove, so changing the code from removeChild to remove is causing the issue.

I confirmed removing the `remove()` method does allow the export to process properly.

#### Product version
v8.2.2

#### Affected browser(s)
IE11

**Comment by PaulDalek** [comment]
Hi @hudson-gev-art

Thank you for contacting us and reporting this issue. I tried to export a chart from the provided demo on IE11 (v11.0.9600.19867), on Windows 8.1 and it looks like everything works as expected. Chart exports and there is no error. Have you tried to run the demo without any editor (for example placing all in the simplified HTML file) and check if the error occurs? Also, could you please provide me with the exact IE version that you use and under what OS? Thanks.

**Comment by hudson-gev-art** [comment]
I looked closer and I should have mentioned we are using styledMode. 

So I've added a [fiddle](https://jsfiddle.net/__hudson__/xL32tfu7/10/) to add styledMode to the demo. I copied that fiddle into a simplified HTML file as requested and was able to trigger the issue. 

```
Object doesn't support property or method 'remove'
```

I clicked on each Download link for each download option (PNG, JPEG, PDF, SVG) and triggered the same error.
